Денис Павлов

Stackopenflow opened the pull request:
Основы Python. 
Оставшийся Блок 2.

Commit:
Add files via upload

Commit:
Add files via upload

ghost commented:
NumPy Блок 1

Commit:
Add files via upload

NumPy Блок 2 Decimal

Commit:
Delete 1d numpy arrays.py

Commit:
Add files via upload

ghost commented:
NumPy Блок 2 Decimal

vongostev commented on Numpy/1d numpy arrays DECIMAL.py:
```diff
@@ -0,0 +1,64 @@
+
+
+# Дисперсия случайной величины
+def dispersion(P):
```
Это написано плохо. Надо делать более абстрактные функции, то есть на вход подается массив, на выходе число. Не только для распределения Пуассона. Посмотрите определение дисперсии и переделайте.

vongostev commented:
Первое задание принимаю, остальные пока не могу

Commit:
Decimal

Был добавлен третий аргумент для функции, так как в общем случае распределение может быть по любым величинам, а не только натуральным.

Commit:
Create Gravity

Commit:
Гравитация

vongostev commented:
Это типичный пример спагетти-кода. Он плохо структурирован, выполнение размазано по одной гигантской функции. Зачем три типа объектов нужно мне лично непонятно. Хватит одного, а интенсивность взаимодействия должна регулироваться отношением масс. 
Я, конечно, зачет поставлю, потому что формально задание выполнено, но по факту там нормально не использовалось практически ничего из того, что мы разбирали на занятиях.
